docker/api/network.py

```diff
@@ -46,9 +46,10 @@ def inspect_network(self, net_id):
         return self._result(res, json=True)
 
     @check_resource
-    @minimum_version('1.21')
+    @minimum_version('1.22')
```

Comment:
This isn't correct, the function needs to stay at `1.21`. You can add an `if` branch within the function to require a higher version for specific params (there are examples of this already).